app/src/main/java/com/mikn/bestlunch/MapsActivity.kt

```diff
@@ -51,8 +53,13 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     }
 
     override fun onMapReady(googleMap: GoogleMap) {
-        mMap = googleMap;
+        mMap = googleMap
         mMap.addMarker(MarkerOptions().position(LatLng(latitude, longitude)).title("Current Location"))
+        mMap.setOnMarkerClickListener {
+            val list = it.tag as List<String>
+            Log.d("Test", list[1])
+            return@setOnMarkerClickListener true
+        }
```

Comment:
`List`を直接Restクラスを持っておいて, 
`tag`に`List`のインデックスを渡すと, 

```suggestion
        mMap.setOnMarkerClickListener {
            val index= it.tag.toInt()
            val rest: Rest = list[index]
            Log.d("Test", "Info: ${rest.title}, ${rest.tel}")
            return@setOnMarkerClickListener true
        }
```
の`rest.title` のように後で見返したときにわかりやすいコードになる. (ただここまでちゃんと実装できているのは強い)